File: Excel/513 Challenge.R
library(tidyverse)
library(readxl)
library(stringi)

path = "513 Sort by Unit Digit.xlsx"
input = read_excel(path, range = "A1:A10")
test  = read_excel(path, range = "B1:B10")

result = input %>%
  arrange(str_sub(Numbers, -1)) 

identical(result$Numbers, test$`Answer Expected`)
# TRUE

File: Excel/500-599/513/513 Challenge.py
import pandas as pd

path = "513 Sort by Unit Digit.xlsx"
input = pd.read_excel(path, usecols = "A")
test  = pd.read_excel(path, usecols = "B")

result = input.copy()
result['sort'] = result['Numbers'].apply(lambda x: x % 10)
result = result.sort_values(by = 'sort').drop(columns = 'sort').reset_index(drop = True)
print(result['Numbers'].equals(test['Answer Expected'])) # True